test-app/tests/helpers/qunit-module-for.js

```diff
@@ -20,8 +21,8 @@ export default function qunitModuleFor(testModule) {
     },
     afterEach(assert) {
       return testModule.teardown(assert).finally(() => {
-        Ember.Test.adapter = null;
-        if (Ember.testing) {
+        setAdapter(null);
+        if (isTesting()) {
           throw new Error(
             'should not have Ember.testing === true after tests have finished'
```

Comment:
```suggestion
            'should not have isTesting() === true after tests have finished'
```

Reply:
I'm not entirely sure this file is used.

Will investigate in follow up